modules/post/windows/gather/get_bookmarks.rb

```diff
@@ -0,0 +1,68 @@
+class MetasploitModule < Msf::Post
+  include Msf::Post::File
+  include Msf::Post::Windows::UserProfiles
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Bookmarked Sites Retriever',
+        'Description' => %q{
+          This module discovers information about a target by retrieving their bookmarked websites on Google Chrome, Opera and Microsoft Edge.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [ 'jerrelgordon'],
+        'Platform' => [ 'win' ],
+        'SessionTypes' => ['meterpreter'],
```

Comment:
Can this also work with shell payloads?

Reply:
No:
`Post failed: NoMethodError undefined method `fs' for #<Session:shell`